android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/navigation/NavGraph.kt

```diff
@@ -47,7 +48,15 @@ fun NavGraphBuilder.mainGraph(navController: NavController) {
             survey,
             onBackClick = {
                 navController.navigateUp()
+            },
+            onStartSurveyClick = {
+                navController.navigate(AppDestination.SurveyQuestions)
             }
         )
     }
+    composable(AppDestination.SurveyQuestions) {
+        SurveyQuestionsScreen(onCloseClick = {
+            navController.popBackStack()
```

Comment:
Same, `navigateUp`

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/114/commits/0d0b04b5795c2cf45387030653649ef22f39b1ef